cirq-core/cirq/sim/state_vector_simulator.py

```diff
@@ -81,6 +81,20 @@ def _create_simulator_trial_result(
             params=params, measurements=measurements, final_step_result=final_step_result
         )
 
+    def sample_from_amplitudes(
```

Comment:
What's this for ?

Reply:
It says in the comment: it passes the simulator seed into `SimulatesAmplitudes.sample_from_amplitudes`.

This is required because the prng used by other simulator classes isn't available in `SimulatesAmplitudes`; the only alternative is manually passing in the simulator seed, but that conflicts with the design for all of our other simulator methods.

Reply:
So I'm a little worried that having this Amplitude specific function present in `state_vector` might lead some users to it if they are just after traditional state vector sampling techniques (which are way faster if we have the full state vector). Could we somehow add a prng field to SimulatesAmplitudes instead of having to setup this forwarding so we can keep the changes entirely in SimulatesAmplitudes ?

Reply:
Discussed offline - removed the `StateVectorSimulator` method.